[Fix-17527][registry-api] Fix Master Worker API startup fails to clean failover finished nodes due to invalid ZooKeeper path

qiong-zhou opened the pull request:
Master Worker API startup fails to clean failover finished nodes due to invalid ZooKeeper path

<!--Thanks very much for contributing to Apache DolphinScheduler, we are happy that you want to help us improve DolphinScheduler! -->

## Purpose of the pull request

fix #17527

Fix Master Worker API startup fails to clean failover finished nodes due to invalid ZooKeeper path.

## Brief change log

<!--*(for example:)*
- *Add maven-checkstyle-plugin to root pom.xml*
-->
- *Modified org.apache.dolphinscheduler.registry.api.RegistryClient*

## Verify this pull request

<!--*(Please pick either of the following options)*-->

This pull request is code cleanup without any test coverage.

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

qiong-zhou committed:
[Fix] fix issue #17527
Master Worker API startup fails to clean failover finished nodes due to invalid ZooKeeper path

qiong-zhou committed:
[Fix] fix issue #17527
Master Worker API startup fails to clean failover finished nodes due to invalid ZooKeeper path

qiong-zhou committed:
Merge branch 'dev' into Fix-17527

ruanwenjun reviewed (dismissed):
LGTM

SbloodyS reviewed (commented):
Please run `mvn spotless:apply` to fix CI. @qiong-zhou

qiong-zhou committed:
[Fix] fix issue #17527
Master Worker API startup fails to clean failover finished nodes due to invalid ZooKeeper path

qiong-zhou commented:
@SbloodyS Sorry for my mistaken, code format has been corrected now, please check.

SbloodyS reviewed (approved):
+1

ruanwenjun reviewed (approved):
LGTM
